### DISCUSSION TITLE
Broken freetube

### DISCUSSION THREAD

**Discussion by Nothoryus025** [FreeTubeApp/FreeTube#5509] [General]
I've been seeing some updates saying that this program, which was great, has now become obsolete and outdated, it no longer loads videos, it gives errors in the Local API and the Individous API too, it stopped supporting downloads on 720p some time ago and the only good thing left was watching the videos without advertisements, but now that's not even possible, it has become real rubbish on my computer, unfortunately, and there's no point in it I'm a fanboy of the program and I want to disagree with my opinion, you should be worried in fixing the project that "was" very beautiful and incredible, but has now become something useless
![Captura de imagem_20240801_113841](https://github.com/user-attachments/assets/bf611474-99a8-424c-a59f-44fe7d16451a)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Dont be so dramatic. FT isnt going anywhere soon.

See all my updates that i provided on https://github.com/FreeTubeApp/FreeTube/issues/5502

**Comment by absidue** [comment]
As mentioned in other discussions and issues (which you clearly didn't check before opening this one), the 720p audio+video stream disappearing was a YouTube change, they stopped offering that stream, there is nothing that we on the FreeTube side can do about it. We can't offer something to download that doesn't exist.
